backend/upload/views.py

```diff
@@ -0,0 +1,18 @@
+from django.conf import settings
+from django.http import HttpResponseForbidden, JsonResponse
+
+from upload.models import Upload
+
+
+def file_upload(request):
+    if request.method != "POST":
+        return HttpResponseForbidden()
+
+    file = request.FILES["file"]
+    upload = Upload(file=file)
+    upload.save()
+    if not settings.USE_AWS_S3:
```

Comment:
Why the difference?

Reply:
Because with S3 storage `upload.file.url` gives me the complete url (ex: `"https://pycon-static.s3.amazonaws.com/media/tmp/sample.png"`) and it's not fine when I have to retrieve in the `UpdateImageForm` with `default_storage.open(url)` because it wants the relative path, this: `'tmp/sample.png'` otherwise the file will not be found! 

And when the file is stored in the file system I need the absolute path because `upload.file.url` gives me this: `'/media/tmp/sample_sHbAJpO.png'` with the `/media` directory included... Really, the file is in `MEDIA_ROOT` + `tmp/sample_sHbAJpO.png`.